routemaster/controllers/topics.rb

```diff
@@ -40,6 +40,7 @@ class Topics < Sinatra::Base
         end
 
         begin
+          raise ArgumentError, "timestamp is from the future" if data['timestamp'].to_i > Routemaster.now
```

Comment:
Given we're doing data validation in `Routemaster::Models::Event`, maybe it'd make sense to put that there?